src/test/java/uk/gov/ons/census/casesvc/testutil/DataUtils.java

```diff
@@ -78,6 +79,28 @@ public static ResponseManagementEvent getTestResponseManagementRefusalEvent() {
     return managementEvent;
   }
 
+  public static ResponseManagementEvent getTestResponseManagementFulfilmentRequestedEvent() {
+    ResponseManagementEvent managementEvent = getTestResponseManagementEvent();
+
+    EventDTO event = managementEvent.getEvent();
+    event.setType(EventType.REFUSAL_RECEIVED);
```

Comment:
This should be set to `FULFILMENT_REQUESTED`

Reply:
I should pay more attention when copy and pasting